### ISSUE TITLE
Rename add-on?

### ISSUE THREAD

**Issue by pdehaan** [mozilla/blok#143]
Asked in IRC, but...
Per changes in #140, are we going to rename the add-on from blok-1.0rc1.xpi to something else?

For example: blok-* to tracking-protection-*
And on that note, rename the repo name to match? Now is a good time to finalize any renaming of repos and addons.

**Comment by pdehaan** [comment]
Possibly related to/dupe of #101, although more specifically about the XPI naming now that we're tagging it as rc1.

**Comment by groovecoder** [comment]
Hmm ... on further consideration, changing the name will mean changing:

uuid in manifest.json (and therefore addons.mozilla.org too)
repo name
file names

So it's a more significant change that I first thought. Which is fine if we're sure "Tracking Protection" is the final name? I'm not sure mozilla/tracking-protection is a good name-space to claim for this ... ? @johngruen ?

**Comment by johngruen** [comment]
Nope, let's not. We're dealing with some e10s related insanity now, so keeping things as is is safer.

**Comment by groovecoder** [comment]
Removed from launch milestone.
